provider/vsphere/client.go

```diff
@@ -200,12 +243,23 @@ func (c *client) Refresh(v *mo.VirtualMachine) error {
 }
 
 func (c *client) getVm(name string) (*mo.VirtualMachine, error) {
```

Comment:
Why not switch the name of this func simply ```vm``` instead of ```getVm```. In go, no notions of **getters** and **setters** exist, like in other languages for example, Java. A more Idiomatic way is:
When you are making a "getter" for a **thing** is much nicer to name it ```thing``` rather than ```getTing``` If you want to make a set function that alternates the state of **thing** name it ```setTing```.


More on this subject: https://golang.org/doc/effective_go.html#Getters